documentation/source/simulator_support.rst

```diff
@@ -25,6 +25,9 @@ In order to use this simulator, set :make:var:`SIM` to ``icarus``:
 
     make SIM=icarus
 
+.. note::
+    This section requires `installation <https://iverilog.fandom.com/wiki/Installation_Guide>`_ of `Icarus Verilog <http://iverilog.icarus.com/>`_.
```

Comment:
"This section" sounds a bit odd to me: it's not this section of text that needs Icarus Verilog, but running a design with it. How about (something like) "A working installation of Icarus Verilog is required. Refer to their documentation for installation instructions."